### ISSUE TITLE
Rename config.toml to hugo.toml and update quickstart guide

### ISSUE THREAD

**Issue by sanchpet** [jpanther/lynx#154]
### What happened?

I am completely new to Hugo and several days ago I started using Lynx theme and followed quickstart guide. 

After completing the 3rd step (start server with `hugo server`), on the 4th step I've noticed that there was no `config.toml` in the root folder of my website, but there was a `hugo.toml` file. 
So I just replaced `hugo.toml` with the contents of yours `config.toml` and the website started perfectly.

Did I miss something or the filename should be changed? As I see, the creators of Hugo [started using](https://github.com/gohugoio/hugo/issues/8979) `hugo.toml` instead of `config.toml` a long time ago.

### Theme version

v1.4.0

### Hugo version

v0.127.0

**Comment by nicolascl86** [comment]
Hey there, I am so sorry to be getting back to you so late. To be honest I am completely new to everything with programming so I am so overwhelmed with this Cloud Resume Challenge I decided to do. I am so overwhelmed so much so my initial thought when I read your message was that I broke Hugo :-)

Does it still work? Because I am stuck somewhere between Step 3 and 4 as well. I wanted to use another theme I found on the Hugo Website but the instructions don't make it clear where to an amateur like myself as to where to I download the theme, replace the ananke theme and any others changes I have so make. I am still trying to figure it out.
